frontend/src/components/search/SearchSortBy.tsx

```diff
@@ -52,11 +45,13 @@ export default function SearchSortBy({
     },
   ];
 
-  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
-    const newValue = event.target.value;
-    updateTotalResults(totalResults);
```

Comment:
there's no need to set total results in multiple places in the app, since both of these components are responding to the same request